CHANGELOG.md

```diff
@@ -1087,7 +1087,10 @@ activity and on its child transactions (which can be actuals, refunds, and adjus
 - Replace code references to `uk_dp_named_contact` with `uk_po_named_contact`, including database migration
 - Remove "PO Definition" from "Aims/Objectives" CSV headers
 
-[unreleased]: https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/compare/release-115...HEAD
+## [unreleased]
+
+[unreleased]: https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/compare/release-116...HEAD
+[unreleased]: https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/compare/release-115...116
```

Comment:
- `[release-116]` at the start
- `...release-116` at the end

Reply:
Oops, well spotted, thanks!

Re: updating Node, I'd say it's more of a dev-specific thing, so not something we'd put in the changelog as it shouldn't be something that affects users.